css/cartList.css

```diff
@@ -0,0 +1,143 @@
+@import url("/styles.css");
+
+.main-wrapper{
+    display: flex;
+    flex-direction: column;
+}
+
+.main-wrapper h1{
+    margin: 1rem;
+    align-items: center;
+    justify-content: center;
+}
+
+.cart-wrapper{
+    display: grid;
+    grid-template-columns: 2.5fr 1.25fr;
+    margin: 0 2rem;
+    grid-column-gap: 1rem;
+}
+
+.cart-header{
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin: 1rem 0;
+    padding: 1rem;
+}
+
+.cart-content{
+    flex-grow: 1;
+    display: grid;
+    grid-template-columns: repeat(auto-fit, minmax(260px, 1fr));
+    margin: 1rem;
+    padding: 1rem;
+}
+
+.card-comp{
+    width: auto;
+}
+
+.footer-card{
+    display: flex;
+    flex-flow: row wrap;
+    align-items: center;
+    justify-content: space-between;
+    padding: 5px;
+}
+
+.btn-addToCart{
+    margin: 0.625rem;
+    background-color: #FFA500;
+    padding: 0.5rem 1rem;
+    font-weight: 800;
+    color: white;
+    border: transparent;
+    border-radius: 0.5rem;
+}
+
+.card-dismiss i{
+    color: #FF0000;
+}
+
+.order-header{
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    margin: 2rem 0;
+}
+
+.order-summary-card{
+    display: flex;
+    flex-direction: column;
+    justify-content: space-around;
+    border: 1px solid var(--headerbg);
+    height: 16.45rem;
+    padding: 1rem;
+    color: #6b7280;
```

Comment:
for colors, you can use variables here

Reply:
It was just used once, so used it explicitly